src/components/bookStoreApp.jsx

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route, Navigate } from 'react-router-dom';
+import Books from './Books';
+import Categories from './Categories';
+import Navbar from './Navbar';
+import NotMatch from './NotMatch';
+import styles from '../styles/bookStoreApp.module.css';
+
+const BookStoreApp = () => (
+  <div className={styles.pageContainer}>
+    <Routes>
+      <Route path="/" element={<Navbar />}>
+        <Route path="/Books" element={<Books />} />
+        <Route path="/" element={<Navigate to="/Books" />} />
+        <Route path="/Categories" element={<Categories />} />
+        <Route path="*" element={<NotMatch />} />
+      </Route>
+    </Routes>
```

Comment:
- [ ] This code should be present in the `App` component directly, kindly make sure to remove this component and use the `App` component to set up Routes